frontend/compute-system/src/components/AppStore/AppRating/interfaces/appRating.ts

```diff
@@ -0,0 +1,4 @@
+export default interface AppRating {
```

Comment:
use Prettier as code formatter. It does the job quite nice!
And then resave all PR files :D